### PR TITLE
dotCMS/core#26310 Move `DotFieldValidationMessageComponent` to UI lib

### LINKED ISSUE #26310: [UI] Move `DotFieldValidationMessageComponent` to UI lib

### PR DESCRIPTION
**Pull request by oidacra**
### Proposed Changes
* Move `DotFieldValidationMessageComponent` to UI lib

### Checklist
- [x] Tests
- [ ] Translations
- [ ] Security Implications Contemplated (add notes if applicable)

### <samp>🤖 Generated by Copilot at a1fdb68</samp>

Removed `DotFieldValidationMessageModule` and replaced it with `DotFieldValidationMessageComponent` in various modules and tests. This simplifies the code structure, avoids circular dependencies, and improves performance and readability. Deleted unused files related to the removed module.

## Related Issue
Fixes #26310 

# Explanation of Changes

### <samp>🤖 Generated by Copilot at a1fdb68</samp>

*  Replace `DotFieldValidationMessageModule` with `DotFieldValidationMessageComponent` from `@dotcms/ui` to avoid circular dependencies and use the component directly ([link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-163a75efd568c07856a7d9562d4d2ab070da66b90e75e5e2907c814380175e7cL31-R33), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-163a75efd568c07856a7d9562d4d2ab070da66b90e75e5e2907c814380175e7cL54-R53), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0ae6b209d7cf2d77c4d1cb812b6fbdb4a0d8bd0f0ff7f09e3c367ba6f7f0d21dL13), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0ae6b209d7cf2d77c4d1cb812b6fbdb4a0d8bd0f0ff7f09e3c367ba6f7f0d21dL19-R23), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0ae6b209d7cf2d77c4d1cb812b6fbdb4a0d8bd0f0ff7f09e3c367ba6f7f0d21dL137-R140), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-e956270ba3bab9c8e69ee0af7020626f5819bb0fc0142e067fb03967e3f8df04L8-R17), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-e956270ba3bab9c8e69ee0af7020626f5819bb0fc0142e067fb03967e3f8df04L26-R30), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a6dabcf3caf1a5d3bcf02b6472fb88e401c76705e583cdf7397c8f5d1aba137dL15-R20), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a6dabcf3caf1a5d3bcf02b6472fb88e401c76705e583cdf7397c8f5d1aba137dL120-R123), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-e2f6c63ae07a48a7e475b7a2dd04a42bb9aebd0e78701ee20519f822a6434cf7L8-R14), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-e2f6c63ae07a48a7e475b7a2dd04a42bb9aebd0e78701ee20519f822a6434cf7L20-R23), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c4c77fba3abe168b6fcc536e786ae05dc32bc3f04bd31a2effecc79decab5fb2L30), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c4c77fba3abe168b6fcc536e786ae05dc32bc3f04bd31a2effecc79decab5fb2L44-R43), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c4c77fba3abe168b6fcc536e786ae05dc32bc3f04bd31a2effecc79decab5fb2L215-R214), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c4c77fba3abe168b6fcc536e786ae05dc32bc3f04bd31a2effecc79decab5fb2L487), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c6c86e6444e256c90c940fbcb4401e60458fefddbdfdc68c249d67fbc42f902bL11-R12), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c6c86e6444e256c90c940fbcb4401e60458fefddbdfdc68c249d67fbc42f902bL33-R38), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-1a3fa73a2b385aaa3e8f3691d24ee87d49a4fa95918fb3851d69bc92e5d08630L8-R13), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-1a3fa73a2b385aaa3e8f3691d24ee87d49a4fa95918fb3851d69bc92e5d08630L31-R34), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a9316b9838a151130d4112743cbd198569b6ae9e05e6341f16b453f048904ec2L24), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a9316b9838a151130d4112743cbd198569b6ae9e05e6341f16b453f048904ec2L41-R40), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a9316b9838a151130d4112743cbd198569b6ae9e05e6341f16b453f048904ec2L168-R167), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0f3ec14af7770b7e5376a79d0ebe9857b53fd3a61a4afb3725d1f5f515e95753L21), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0f3ec14af7770b7e5376a79d0ebe9857b53fd3a61a4afb3725d1f5f515e95753R45), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-0f3ec14af7770b7e5376a79d0ebe9857b53fd3a61a4afb3725d1f5f515e95753L133-R133), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-74607c839ef9b9b1e38d64e68cf86d4bb0799df279eba4d2faa353ac506d512bL15), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-74607c839ef9b9b1e38d64e68cf86d4bb0799df279eba4d2faa353ac506d512bL25-R24), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-74607c839ef9b9b1e38d64e68cf86d4bb0799df279eba4d2faa353ac506d512bL106-R105), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a5d1877d61972d27a4b75801d1555308078ac26097ad796d2deea8d709517118L9-R18), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-a5d1877d61972d27a4b75801d1555308078ac26097ad796d2deea8d709517118L26-R30), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-b79d4bf1823fd297b0baf0ac3af3e68a94f6f2d04c0f0964cf8f6f5962cc5d3bL10), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-b79d4bf1823fd297b0baf0ac3af3e68a94f6f2d04c0f0964cf8f6f5962cc5d3bL16-R15), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-b79d4bf1823fd297b0baf0ac3af3e68a94f6f2d04c0f0964cf8f6f5962cc5d3bL58-R57), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-1bdc0e6d0b06a80d46fc1257c50881bd85349b60e07a0ea3acfa962279d88b11L7), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-1bdc0e6d0b06a80d46fc1257c50881bd85349b60e07a0ea3acfa962279d88b11L13-R17), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-1bdc0e6d0b06a80d46fc1257c50881bd85349b60e07a0ea3acfa962279d88b11L26-R30), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c5910d6e1ee00cfcc78b7592088fb9ddc9477a4421d1a99cc16741f1858dd360L12-R16), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-c5910d6e1ee00cfcc78b7592088fb9ddc9477a4421d1a99cc16741f1858dd360L57-R56), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-f9cd5f61b7e6dea45c5ddd24518ebb9ec6b785561eea8059cfa45560f2fe9cabL10-R14), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-f9cd5f61b7e6dea45c5ddd24518ebb9ec6b785561eea8059cfa45560f2fe9cabL27-R25), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-46b6e75614ef2bd3952d6e8806263a18ea9e21090bd13dcd1bfd808e4229159dL13-R15), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-46b6e75614ef2bd3952d6e8806263a18ea9e21090bd13dcd1bfd808e4229159dL30-R30), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-877e1c0db43701b98310899f861f8624c48639eca7ee62a97a34a44bb38e687dL17), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-877e1c0db43701b98310899f861f8624c48639eca7ee62a97a34a44bb38e687dL34-R33), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-877e1c0db43701b98310899f861f8624c48639eca7ee62a97a34a44bb38e687dL146-R145))
* Delete unused files `dot-file-validation-message.module.ts` and `dot-field-validation-message.html` ([link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-3c1f11bd8aae63406630524107d2d1f01912d1e525341a0ba3d30199685d71e6), [link](https://github.com/dotCMS/core/pull/26315/files?diff=unified&w=0#diff-99056f2008da15df1d8b8d951b1aade57a4d8178aac83b80d002e9daf2ef3c05))

### CODE REVIEW

**Review by fmontes — APPROVED** [review]

**Review by zJaaal — APPROVED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts [hunk 1]
```diff
@@ -27,7 +26,6 @@ import { DotExperimentsConfigurationStore } from '../../store/dot-experiments-co
         CommonModule,
         ReactiveFormsModule,
 
-        DotFieldValidationMessageModule,
```
Comment (rjvelazco) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts:30]: Is not needed here?

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.ts [hunk 2]
```diff
@@ -35,7 +34,6 @@ import {
         CommonModule,
         ReactiveFormsModule,
 
-        DotFieldValidationMessageModule,
```
Comment (rjvelazco) [inline core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.ts:38]: Not longer needed?

**Review by rjvelazco — CHANGES_REQUESTED** [review]

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.ts (hunk 2 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-traffic-allocation-add/dot-experiments-configuration-traffic-allocation-add.component.ts:38]: no needed. Now is `Standalone`

File: core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts (hunk 1 above)
Comment (oidacra) [reply core-web/libs/portlets/dot-experiments/portlet/src/lib/dot-experiments-configuration/components/dot-experiments-configuration-scheduling-add/dot-experiments-configuration-scheduling-add.component.ts:30]: no needed. Now is `Standalone`

**Review by jdotcms — APPROVED** [review]

**Review by rjvelazco — APPROVED** [review]

**Pull request merged**